File: arrays/array1.py
def sorted_squared(array):
    new_array =[0] * len(array)
    i = 0
    j = len(array) - 1

    for k in reversed(range(len(array))):
        sq_i = array[i] ** 2
        sq_j = array[j] ** 2
        if sq_i > sq_j:
            new_array[k] = sq_i
            i += 1
        else:
            new_array[k] = sq_j
            j -= 1
    return new_array

a = [-3,-3,7,13, 18]
b = [1, 4, 5, 7]
c = [-11, 1, 4, 5, 9]

print(sorted_squared(a))
print(sorted_squared(b))
print(sorted_squared(c))

File: arrays/array1.js
function sortedSquared(array){
    const newArray = new Array(array.length).fill(0)
    for (let i=0;i<array.length;i++){
        newArray[i] = Math.pow(array[i],2)
    }
    newArray.sort((a,b) => a-b)
    return newArray
}

a = [1, 3, 4, 5, 6]
b = [-1, -5, 6, 5, 4]
c = []

console.log(sortedSquared(a)) // [1, 9, 16, 25, 36]
console.log(sortedSquared(b)) // [1, 16, 25, 36, 25]
console.log(sortedSquared(c)) // [0, 0, 0, 0, 0]
console.log(sortedSquared([0, 0, 0, 0, 0])) // [0, 0, 0, 0, 0]